### PR TITLE
Update get-started-core-ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated license requirement per https://learn.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#how-do-users-benefit-from-the-service-16

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9724

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/get-started-core-ediscovery.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ Licensing for eDiscovery (Standard) requires the appropriate organization subscr
   - A Microsoft 365 E3 or Office 365 E3 license or higher
 
    OR
+   
+  - Microsoft 365 Business Premium (Exchange only)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/get-started-core-ediscovery.md]:
```suggestion
  - Microsoft 365 Business Premium (Exchange only) license
  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by v-tophillips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@yogkumgit Validation has been passed.

The author has already approved the changes

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**